src/bot.py

```diff
@@ -31,7 +32,8 @@ async def bing(ctx):
 
 @bot.command(name="day")
 async def day(ctx):
-    day = datetime.today().strftime("%A")
+    EST = timezone("US/Eastern")
```

Comment:
but what about Swan, who doesn't live in eastern time zone?

Reply:
:TEETH: